services/gitclub/requirements.txt

```diff
@@ -19,21 +19,21 @@ Jinja2==3.1.2
 MarkupSafe==2.1.1
 mypy==0.971
 mypy-extensions==0.4.3
-opentelemetry-api==1.11.1
-opentelemetry-distro==0.30b1
-opentelemetry-exporter-otlp-proto-http==1.11.1
-opentelemetry-instrumentation==0.30b1
-opentelemetry-instrumentation-dbapi==0.30b1
-opentelemetry-instrumentation-flask==0.30b1
-opentelemetry-instrumentation-psycopg2==0.30b1
-opentelemetry-instrumentation-requests==0.30b1
-opentelemetry-instrumentation-sqlalchemy==0.30b1
-opentelemetry-instrumentation-wsgi==0.30b1
-opentelemetry-proto==1.11.1
-opentelemetry-sdk==1.11.1
-opentelemetry-semantic-conventions==0.30b1
-opentelemetry-util-http==0.30b1
-oso-cloud==0.9.0
+opentelemetry-api==1.13.0
```

Comment:
Had to upgrade this because of the `backoff` dependency in `oso-cloud`. And so I had to upgrade all the related opentelemetry dependencies, too.